javascript/testing_javascript/project_battleship.md

```diff
@@ -30,7 +30,7 @@ We have not yet discussed testing the appearance of a webpage, which requires a
    4. The game loop should step through the game turn by turn using only methods from other objects.  If at any point you are tempted to write a new function inside the game loop, step back and figure out which class or module that function should belong to.
    5. Create conditions so that the game ends once one player's ships have all been sunk.  This function is appropriate for the Game module.
 5. Finish it up
-   1. Implement the system that allows players to place their ships. There are several options available for letting users place their ships.  You can let them type coordinates for each ship, or investigate implementing drag and drop.
+   1. Implement the system that allows players to place their ships. There are several options available for letting users place their ships. You can let them type coordinates for each ship, cycle through random placements, or investigate implementing drag and drop.
```

Comment:
I think we should be explicit about using a button here, otherwise, "cycle through random placements" sounds confusing.
```suggestion
   1. Implement the system that allows players to place their ships. There are several options available for letting users place their ships. You can let them type coordinates for each ship, have a button to cycle through random placements, or investigate implementing drag and drop.
```